### ISSUE TITLE
auto collapse in the left pane when delete a notebook

### ISSUE THREAD

**Issue by MatthewLee123** [HappenApps/Quiver#1212]
when expand my notebooks and  then delete any sub notebooks on the left pane,  I find all notebooks collapse automatically. it's not good.  because, if there some other sub notebooks same there I want to delete, I still need to expand all notebooks one by on again and find them.

you can give it try.

**Comment by ylian** [comment]
Yeah this can be improved.

**Comment by MatthewLee123** [comment]
thanks for the prompt response. 
yes, it's actually an old issue.

**Comment by MatthewLee123** [comment]
@ylian Hi, can I know if this issue is fixed on 3.2.3 ?

**Comment by ylian** [comment]
No, as you said, it's an old issue. The whole UI needs to be revamped soon, so I am not keen to fixing issues like this.

**Comment by MatthewLee123** [comment]
Hi, i think you mix this issue with #1213 ?
which was talking about integrating multiple libraries, it definitely needs changing the whole UI.

however, this case is about deleting notebook arouses collapsing all notebooks automaticlly. and you said it can be improved 11 days ago.

the reason  why i said it's an old issue is it existed since many old versions. correct me if i am wrong.

thanks .

**Comment by ylian** [comment]
No, I am not mistaken. This issue has existed since many old versions. The app's UI needs to be revamped (this has nothing to do with #1213). In 3.2, I revamped the model layer. In 3.3, I will revamp the editors. And in 3.4, I will revamp the whole app UI.

The current UI implementation is getting hard to maintain. I am looking for better solutions. Patching UI issues at the moment doesn't make much sense.

**Comment by MatthewLee123** [comment]
Thanks
Actually, it's not a big deal. just a tiny improvement is possible.

Thanks for being aware of this.
